homework/week4/chrispark0304/15649.c

```diff
@@ -0,0 +1,49 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+void print_arr(int arr[], int len){
+    for(int i = 0 ; i < len ; i++){
+        if(i == len - 1){
+            printf("%d\n", arr[i]);
+        }
+        else{
+            printf("%d ",arr[i]);
+        }
+    }
```

Comment:
for문에서 첫 iteration, 마지막 iteration에 if걸기보다 일반화하는 편이 좋습니다. (물론 어쩔 수 없을 때도 있긴합니다)
```suggestion
    for(int i = 0 ; i < len ; i++){
        printf("%d ",arr[i]);
    }
    printf("\n");
```